packages/tupaia-web/src/layout/Sidebar/StaticMap.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import polyline from '@mapbox/polyline';
+import { Position } from 'geojson';
+import { DEFAULT_BOUNDS } from '../../constants';
+import { Media } from './Media';
+
+const areBoundsValid = (b: Position[]) => {
+  return Array.isArray(b) && b.length === 2;
+};
+
+const getLatLongForBounds = (bounds: Position[]) => {
+  const lats = bounds.map(latLong => latLong[0]);
+  const longs = bounds.map(latLong => latLong[1]);
+  const minLat = Math.min(...lats);
+  const maxLat = Math.max(...lats);
+  const minLong = Math.min(...longs);
+  const maxLong = Math.max(...longs);
+
+  const latitude = minLat + (maxLat - minLat) / 2;
+
+  let longitude = minLong + (maxLong - minLong) / 2;
+
+  // Mapbox static api throws errors with anything over 180.
+  longitude = longitude > 180 ? 180 : longitude;
+
+  return { latitude, longitude };
+};
+
+const MAPBOX_TOKEN = import.meta.env.REACT_APP_MAPBOX_TOKEN;
+const MAPBOX_BASE_URL = 'https://api.mapbox.com/styles/v1/sussol/cj64gthqq297z2so13qljil5n/static';
+
+const makeStaticMapUrl = (polygonBounds: Position[]) => {
+  const polygonPoints = [
+    [polygonBounds[1][0], polygonBounds[1][1]],
+    [polygonBounds[1][0], polygonBounds[0][1]],
+    [polygonBounds[0][0], polygonBounds[0][1]],
+    [polygonBounds[0][0], polygonBounds[1][1]],
+    [polygonBounds[1][0], polygonBounds[1][1]],
+  ];
+
+  const encodedPolyline = encodeURIComponent(
+    polyline.encode(polygonPoints as Array<[number, number]>),
+  );
+
+  const showBox = polygonBounds !== DEFAULT_BOUNDS;
```

Comment:
I have a feeling using `!==` with arrays won't work, and would potentially always be `false`... Might need to use `.every()` or compare the stringified arrays here

Reply:
Yes you are absolutely right. I copied this from the old site and didn't think about it that much and so after your review comment I was confused about why it works on the old site. I think it's because the polygonBounds 
 are set using the actual DEFAULT_BOUNDS const upstream so the equality does actually work which is just lucky and pretty dodgy. I ended up doing the array comparison just by using JSON.stringify which works well for small arrays as long as there are no nested functions which fits this use case.